proof/crefine/RISCV64/Arch_C.thy

```diff
@@ -1322,6 +1322,105 @@ lemma performPageInvocationMapPTE_ccorres:
   apply clarsimp
   done
 
+(* FIXME move to SR_Lemmas_C *)
+lemma vmRightsToBits_bounded:
+  "vmRightsToBits rights < 4"
+  by (cases rights; clarsimp simp: vmRightsToBits_def)
+
+(* FIXME move to SR_Lemmas_C *)
+lemma vmRightsToBits_not_0:
+  "vmRightsToBits rights \<noteq> 0"
+  by (cases rights; clarsimp simp: vmRightsToBits_def)
+
+(* FIXME move to SR_Lemmas_C *)
+lemma vmRightsToBits_vmrights_to_H:
+  "\<lbrakk> rights < 4; rights \<noteq> 0 \<rbrakk> \<Longrightarrow> vmRightsToBits (vmrights_to_H rights) = rights"
+  apply (clarsimp simp add: vmrights_to_H_def vm_rights_defs vmRightsToBits_def split: if_splits)
+  apply (drule word_less_cases, erule disjE, simp, simp)+
+  done
```

Comment:
I know you just moved them inside the file, but if possible, please move them up to `SR_Lemmas_C` to resolve the FIXMEs.